src/Config/Output.php

```diff
@@ -168,7 +168,11 @@ public function getInputBOM()
     abstract public function getIterator();
 
     /**
-     * Outputs all data on the CSV file
+     * Outputs all data on the CSV file. 
+     * The CSV data  will be written to the output buffer. Returning 
+     * the result of this function (or anything else) may append 
```

Comment:
This description adds more confusion ... there's not **may** !! The method signature [is very strict](http://csv.thephpleague.com/basic-usage/#output)